platform/eureka/controller.go

```diff
@@ -0,0 +1,72 @@
+package eureka
+
+import (
+	"time"
+
+	"reflect"
+
```

Comment:
Yep, they got added randomly. Surprised that goimports doesn't remove them automatically.